### PR TITLE
Merged main into dev 1.29 branch

### PR DESCRIPTION
**Pull request by Princesso**
This PR syncs the dev-1.29 branch with main to keep the dev-1.29 branch up to date.

cc: @katcosgrove @taniaduggal @drewhagen @harshitasao

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
/approve

**Comment by sftim** [comment]
/assign palnabarun

for steering approval (this merges a banner change)

**Comment by Princesso** [comment]
/assign @pacoxu 
/assign @pohly 

Anyone from the steering committee can review this PR. Thanks.

File: data/announcements/scheduled.yaml
Comment (pohly) [inline data/announcements/scheduled.yaml:198]: +1 from me on behalf of @kubernetes/steering-committee 

Strictly speaking, we need a majority before the final approval.

Comment (sftim) [reply data/announcements/scheduled.yaml:198]: You sure @pohly ? The same commit was already approved for _main_.

Comment (pohly) [reply data/announcements/scheduled.yaml:198]: We discussed it internally and concluded that voting would be more consistent with the [charter](https://github.com/kubernetes/steering/blob/main/charter.md#voting), even for routine business. As a new member of the committee I don't feel comfortable with deviating from that - others might see this differently.

Comment (Princesso) [reply data/announcements/scheduled.yaml:198]: Thanks for the feedback @palnabarun @pacoxu could you please have a look to decided on this for final approval?

Comment (soltysh) [reply data/announcements/scheduled.yaml:198]: :+1: from me as well

Comment (mrbobbytables) [reply data/announcements/scheduled.yaml:198]: +1 from me. IMO, I think in this instance where it is just to keep the branches in sync and we've already approved it in main, it should be okay to move forward without majority consensus.  It's more an edge case that we'll only ever really see in use on these types of PRs that will come up every release.

Comment (Princesso) [reply data/announcements/scheduled.yaml:198]: Awesome, thank you all for your feedback. If there is no objection, can any of you please approve? Thanks.

Comment (palnabarun) [reply data/announcements/scheduled.yaml:198]: +1 from steering to complete the loop on this.

Comment (Princesso) [reply data/announcements/scheduled.yaml:198]: Thanks @palnabarun. Could you please use /approve to highlight your approval? Thanks.

**Comment by Princesso** [comment]
@caesarxuchao @bgrant0607 could you kindly review this PR and give your approval if everything looks good to you? Warm regards.

**Review by palnabarun — APPROVED** [review]
We have 4 steering votes.

/approve

**Comment by sftim** [comment]
Hmm, I wonder what EasyCLA is playing at.

**Comment by katcosgrove** [comment]
CI appears to be haunted

**Comment by justaugustus** [comment]
/approve (Steering)

_nudges bots_:
/check-cla

**Comment by justaugustus** [comment]
While we're waiting... how about a:
/joke

**Comment by tengqm** [comment]
Just out of curiosity ... since when the sync from main to a dev-1.xx branch needs an upcall to the steering committee, and even a quorum from the committee? The committee has no other important works to oversee or we are really trying to improve something? I can smell something bad from this based on what I witnessed somewhere else. A truly open community encourages all open contributions, open discussions and open collaborations. Redundant rules. policies or processes are invented and enforced only when there are no other choices. If there is a trend towards more bureaucracy rather than more productivity, please let us watch out and think twice. Does this PR really worth your precious time?

**Comment by katcosgrove** [comment]
We do not usually need steering involvement to merge branch syncs. This PR requires it because it merges a site banner, and those require approval from steering. Please be kind.

**Comment by tengqm** [comment]
> We do not usually need steering involvement to merge branch syncs. This PR requires it because it merges a site banner, and those require approval from steering. Please be kind.

Sorry for being ignorance about this exception. I was hoping that we as a community foster more trusts to contributors and motivates them rather than spending time on rules, processes.
I was shocked by this thread because I never saw this before, during my 8 years involvement with the community. I was worrying.

**Comment by Priyankasaggu11929** [comment]
Per the discussion https://kubernetes.slack.com/archives/C1TU9EB9S/p1698411010005219?thread_ts=1698411010.005219&cid=C1TU9EB9S, going ahead and manually merging the PR.

**Pull request merged**
